### PR TITLE
cmd/libsnap-confine-private: move device cgroup files, add helper to deny a device

### PR DESCRIPTION
**Pull request by bboozzoo**
Preparation for reimplementation of snap-device-helper and more device cgroup changes

### CODE REVIEW

File: cmd/libsnap-confine-private/device-cgroup-support.c [hunk 1]
```diff
@@ -136,6 +144,16 @@ int sc_device_cgroup_allow(sc_device_cgroup *self, int kind, int major, int mino
     return 0;
 }
 
+int sc_device_cgroup_deny(sc_device_cgroup *self, int kind, int major, int minor) {
+    if (kind != S_IFCHR && kind != S_IFBLK) {
```
Comment (mvo5) [inline cmd/libsnap-confine-private/device-cgroup-support.c:148]: Thanks for this new check

**Review by mvo5 — APPROVED** [review]
Thank you. The first two commits just move code around and the changes in the last one look fine.

**Review by stolowski — APPROVED** [review]
LGTM

**Review by mardy — APPROVED** [review]
LGTM!

**Review by alexmurray — APPROVED** [review]
LGTM!

**Pull request merged**